src/pages/en/guides/styling.md

```diff
@@ -2,16 +2,29 @@
 layout: ~/layouts/MainLayout.astro
 title: Styling & CSS
 description: Learn how to style components with Astro.
+setup: |
+  import Since from '../../../components/Since.astro';
 ---
+Astro includes special handling to make writing CSS as easy as possible. You can use your favorite libraries, write your own css files, or take advantage of Astro's own scoped styling.
 
-Astro includes special handling to make writing CSS as easy as possible. Styling inside of Astro components is done by adding a `<style>` tag anywhere.
+## Styling an Astro Component
 
-## Astro component styles
+Astro allows you to include both global and scoped styles right inside an Astro component. You can write plain CSS (or use a CSS language extension) directly in Astro's `<style>` tag to style your component.
 
-By default, all Astro component styles are **scoped**, meaning they only apply to the current component. This can be very easy to work with, as you only have to worry about what’s in your current document at any given time.
+By adding an [integration](/en/integrations/integration-guide) like Tailwind, you can also take advantage of writing utility classes inline in your Astro component template.
```

Comment:
This is throwing a lot of concepts at the reader for an introduction to styling. Lets see if we can lighten it up.

What about:

```diff
---

Astro was designed for familiar, powerful styling. Write your own CSS directly inside of an Astro component or import your favorite CSS library like [Tailwind](#TODO). Advanced styling languages like [Sass](#TODO) and [Less](#TODO) are also supported.

## Styling in Astro

The easiest way to style your Astro component is with a basic `<style>` tag. When you place a `<style>` tag in your component template, Astro knows to build, optimize and scope your styles for you, automatically.

'''
TODO: SUPER SIMPLE EXAMPLE
Like, just a style tag in a basic component. 
No need to make the distinction between scoping and global just yet.
'''

### Scoped Styles

- Styling inside of Astro components is done by adding a `<style>` tag in the component template.

By default, all Astro component styles are **scoped** // ...
```